explorer-api/src/mix_nodes/models.rs

```diff
@@ -153,6 +154,11 @@ impl ThreadsafeMixNodesCache {
         let denom = &node.mixnode_details.original_pledge().denom;
         let rewarding_info = &node.mixnode_details.rewarding_details;
 
+        let mut operating_cost = rewarding_info.cost_params.interval_operating_cost.clone();
+        if operating_cost.denom == "unym" {
```

Comment:
isn't the if redundant? it should always be returned in either `unym` or `nym`

Reply:
No? `operating_cost` is of type Coin, which can be either "nym" or "unym" values (as far I understand it). So the logic is to convert only when it's "unym", right?